packages/utils/functions/axios/createAxiosInstance/index.ts

```diff
@@ -55,7 +64,21 @@ export const createAxiosInstance = ({
       }
       return returnData && response.data ? response.data : response
     },
-    (error) => {
+    async (error) => {
+      if (
+        tokenType === TokenTypes.jwt &&
+        error.response?.status === 401 &&
+        error.config.url !== REFRESH_TOKEN_URL
+      ) {
+        try {
+          const originalRequest = error.config
+          // eslint-disable-next-line @typescript-eslint/no-use-before-define
+          return await refreshAccessToken(cookieName, refreshCookieName, originalRequest)
+        } catch (refreshError) {
+          return Promise.reject(refreshError)
```

Comment:
call endpoint to logout user and test if redirects user to login page

Reply:
I have an implementation of this already, but I just realized that we would need to add nextjs as an implementation of the auth package to use the router and I'm not sure if that's ok. Will discuss it with the rest of the team.

Reply:
btw Should we use some event / pub-sub mechanism to fire of an event from axios interceptor that can be subscribet to in the authentication package to then trigger calling a `logout()`?

Reply:
Made it like that, it now triggers an event, but put the actual subscribing and redirecting into the FE templates, bec we don't have next as a dependency even in the auth package. Not sure if we want to keep it like that for now. Maybe we can have a different package for next stuff

Reply:
Here's the PR: https://bitbucket.org/silverlogic/baseapp-frontend-template/pull-requests/17